src/services/index.ts

```diff
@@ -1 +1,2 @@
 export {default as LogService} from './LogService';
+export {HapticMethods, triggerHaptic} from './HapticService';
```

Comment:
Export the service as a whole, otherwise `services/index` fill will export a bunch of unrelated methods from various services.